common/repositories/template/base_repository.template

```diff
@@ -291,6 +291,32 @@ public:
 		return (results.Success() ? results.RowsAffected() : 0);
 	}
 
+	static int64 GetMaxId(Database& db)
+	{
+		auto results = db.QueryDatabase(
+			fmt::format(
+				"SELECT MAX({}) FROM {}",
+				PrimaryKey(),
+				TableName()
+			)
+		);
+
+		return (results.Success() ? strtoll(results.begin()[0], nullptr, 10) : 0);
```

Comment:
Need to validate `results.begin()[0]` isn't null. Even if the query is a success it might be on some empty state table and return a null result